modules/solid_mechanics/include/bcs/InteractionIntegralBenchmarkBC.h

```diff
@@ -0,0 +1,59 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef INTERACTIONINTEGRALBENCHMARKBC_H
+#define INTERACTIONINTEGRALBENCHMARKBC_H
+
+#include "PresetNodalBC.h"
+#include "CrackFrontDefinition.h"
+
+//Forward Declarations
+class InteractionIntegralBenchmarkBC;
+class Function;
+
+template<>
+InputParameters validParams<InteractionIntegralBenchmarkBC>();
+void addInteractionIntegralBenchmarkBCParams(InputParameters& params);
+
+/**
+ * Implements a boundary condition that enforces rotational displacement around
```

Comment:
I think this comment is a copy/paste for the class you copied this from.  Just update this comment (and check the others), and I'm good with this PR.